capstone/src/main/webapp/script.js

```diff
@@ -224,7 +225,31 @@ async function loadCalendar () {
   if (json['club']['calendar'].length != 0) {
     document.getElementById('calendar-element').src = "https://calendar.google.com/calendar/embed?src=" + json['club']['calendar'];
   }
-  document.getElementById('club-name-cal').value = params.get('name');
+  
+  const officerQuery = '/officer?name=' + params.get('name');
+  const officerResponse = await fetch(officerQuery);
+  const officerText = await officerResponse.text(); 
+  const isOfficer = JSON.parse(officerText);  
+  if (isOfficer) {
+    const eventsList = await fetch('/events?name=' + params.get('name'));
+    const eventsJson = await eventsList.json();
+    document.getElementById('events-list').removeAttribute('hidden');
+    const template = document.querySelector('#event-listing');
+
+    for (var event of eventsJson) {
+      template.content.querySelector('.event-name').innerHTML = event.summary;
+      template.content.querySelector('.event-time').innerHTML = new Date(event.start.dateTime.value).toString();
+      template.content.querySelector('.delete-event').setAttribute('onclick', 'deleteEvent("' + event.id + '");');
```

Comment:
nit: I think you can also do ...('.delete-event').onclick = function() { deleteEvent(event.id) }
Just looks a bit cleaner! It's fine if it doesn't work though

Reply:
It was strange - I had to spend a while on this, because a bunch of things that I thought should have worked didn't, and I think this was one of them.